utils.py

```diff
@@ -888,13 +888,24 @@ def check_auth(user, hutch):
             return True
     return False
 
+# checks if ioc is marked as toggleable between defined versions of IOCs
+# schema will be ioc_name:permittedversion1,permittedversion2,etc
 def check_special(ioc, hutch):
```

Comment:
Getting closer. check_special, or check_special_toggle?  Two parameters or three?  vers or version?

Reply:
thought i forced pushed that change just in time. i should have kept this as a draft until i tested it, but greatly appreciate you finding the errors pre run time